### PR TITLE
Conference page 

### PR DESCRIPTION
**Pull request by Addisu87**
I have created a conference page from a mobile-first approach to the desktop version. The pages are the home and about sections only. 
https://www.loom.com/share/727c670763d94b61a06e529362319bf3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+          conference on Astronomy, Astrophysics and Space Science" which is
+          going to be held during June 27-28, 2022 at Addis Ababa, Ethiopia.
+        </p>
+        <h3>June 27-28, 2022 Addis Ababa, Ethiopia</h3>
+        <p class="location">&#64; 4-kilo main hall, Addis Ababa University.</p>
+      </div>
+    </section>
+
+    <!-- Program-section -->
+    <section id="program">
```
Comment (Munsa1) [inline index.html:72]: - [ ] There is no button in the main program section part of the original design. kindly fix it. 
 
Your design
image

Original design
image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
```
Comment (Munsa1) [inline about.html:1]: - The about page does not have a to heading for the mobile view, Kindly fix it

Your design
image

Figma design
image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+              />
+              <span class="cover"></span>
+              <h3>2020</h3>
+              <p>Buenos Aires</p>
+            </a>
+          </div>
+        </div>
+      </section>
+    </main>
+
```
Comment (Munsa1) [inline about.html:157]: - [ ]  Above the footer section is supposed to a partner section for the mobile view, which is missing. Kindly fix it.

Your design
image

Figma design
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+      <h2 class="section-heading">Sessions and Tracks</h2>
+      <div class="items">
```
Comment (Munsa1) [inline index.html:74]: - [ ] The main program section is not properly aligned. Kindly fix it.

Your design
image

Figma design
image


- [ ] Kindly increase the button width aswell inline with the figma design

Your design
image

Figma design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES

Hi @Addisu87  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Gitflow :heavy_check_mark: 
Linters :heavy_check_mark:
Descriptive Readme :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @Addisu87  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**